docs/user-guide/share.md

```diff
@@ -1,32 +1,45 @@
-# Sharing a Map
-***************
+# Sharing Resources
+*******************
 
-Sharing a map works only if you have already saved your map. It allows you to share the direct link of the map or on your favorite social networks or to embed it in your website.
+In [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) it is possible to share a Resource:
 
-From the home page:
+* Directly from MapStore [Homepage](home-page.md) by clicking on the Share button <img src="../img/button/share.jpg" class="ms-docbutton"/> present in each resource toolbar
 
-* **Open** a saved map.
+<img src="../img/share/share_cards.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-From the main menu bar:
+* Inside the resource by clicking on the [Burger Menu](menu-bar.md#burger-menu) <img src="../img/button/sharebm.jpg" class="ms-docbutton" style="max-height:25px;"/> option (only for Maps)
```

Comment:
Inside the resource (Map, Dashbard etc) by clicking on the [Burger Menu](menu-bar.md#burger-menu) <img src="../img/button/sharebm.jpg" class="ms-docbutton" style="max-height:25px;"/> option.